.gitignore

```diff
@@ -1,4 +1,3 @@
 .cpcache/
 .nrepl-port
-pom.xml
-target/
+.rebel_readline_history
```

Comment:
I would prefer to keep `pom.xml` and `target` in the git-ignore

Reply:
What do you expect to create pom.xml or target?

Reply:
`target` and `pom.xml` files usually created when building a JAR / UberJAR. Those don't need to be committed.